js/packages/eyes-storybook/src/utils/prepare-settings.js

```diff
@@ -0,0 +1,10 @@
+function prepareTakeDomSnapshotsSettings({config, options}) {
+  let result = {...options};
+  result.layoutBreakpoints !== undefined ? result.layoutBreakpoints : config.layoutBreakpoints,
+    (result.disableBrowserFetching = config.disableBrowserFetching);
```

Comment:
This looks strange. You probably meant something like this
```suggestion
  result.layoutBreakpoints = result.layoutBreakpoints !== undefined ? result.layoutBreakpoints : config.layoutBreakpoints;
  result.disableBrowserFetching = config.disableBrowserFetching;
```

Reply:
Move fast and break things. 😎 

Fixed in https://github.com/applitools/eyes.sdk.javascript1/pull/1490/commits/18c3ca4d2b1a8e08845aad205d270c216c304389